README.md

```diff
@@ -1 +1,40 @@
-# vet-clinic-database
\ No newline at end of file
+# vet-clinic-database
+

```

Comment:
Please add the project description in the README file. A professional README should have a project title, description, and a set of instructions to use the project locally.